src/developer_config.yaml

```diff
@@ -4,21 +4,29 @@ global:
   table_config: "SingleLine"
   postcode_csv_check: False
   network_or_hdfs: network #whether to load from hdfs or network (local Python)
+years:
+  current_year: 2022 # TODO: put this in the userconfig
+  previous_years_to_load: 2 # TODO: put this in the userconfig
 runlog_writer:
     write_csv: True # Write the runlog to a CSV file
     write_hdf5: False # Write the runlog to an HDF5 file
     write_sql: False # Write the runlog to a SQL database
     display: False # Display the runlog in the terminal
     log_path: "data/log"
 hdfs_paths:
+  root: None
   logs_foldername: "resdev/logs/run_logs"
   snapshot_path: "/ons/rdbe_dev/snapshot-202012-002-fba5c4ba-fb8c-4a62-87bb-66c725eea5fd.json"
+  hist_folder: "ons/rdbe_dev/BERD_V7_Anonymised"
+  masterlist_path: "data/external/ONSPD_NOV_2022_UK.csv"
   postcode_masterlist: "data/external/ONSPD_NOV_2022_UK.csv"
+  history_path: None
 network_paths:
+  root: None
   logs_foldername: "logs/run_logs"
   snapshot_path: "R:/BERD Results System Development 2023/DAP_emulation/spp_snapshots/snapshot-202012-002-fba5c4ba-fb8c-4a62-87bb-66c725eea5fd.json"
   postcode_masterlist: "R:/BERD Results System Development 2023/DAP_emulation/ONS_Postcode_Reference/ONSPD_NOV_2022_UK.csv"
-
+  history_path: 'R:\BERD Results System Development 2023\DAP_emulation\backseries\v1'
```

Comment:
This path points to a different folder than expected. I ran it successfully using this path:
'R:/BERD Results System Development 2023/DAP_emulation/BERD_V7_Anonymised/'

Reply:
Think I have fixed this here: https://github.com/ONSdigital/research-and-development/pull/43/commits/c6164128e9cba2930b7a0864d13c7b2bed933bdf

Reply:
I have fixed this now.